docs/relnotes.md

```diff
@@ -12,6 +12,15 @@ This is the list of changes to Taipy releases as they were published.
     Please refer to the [Migration page](./migration.md) for potential migration paths for your applications
     implemented on legacy Taipy versions.
 
+## Community edition: 2.3
```

Comment:
Taipy Core 2.2 will be shipped in the next two to three weeks.
The develop branch of taipy-doc is the source for the 2.2 documentation set.
This change is going to impact the Release Notes for 2.2, yet referencing 2.3.
I would create a new issue to perform this one change, and work on it *after* 2.2

Reply:
I believe we don't need a new issue. I just blocked the issue until 2.2 is delivered. 
You can just assign the PR to me. Before each release, I am reviewing all the PRs anyway.

Reply:
I'm good with that too 😃